src/components/ui/PopOver/Styled.tsx

```diff
@@ -16,6 +16,8 @@ export const StyledPopOverMenu = styled.ul`
   padding: 0.5rem 0;
   box-shadow: ${(props) => props.theme.popOver.shadow};
   z-index: ${(props) => props.theme.zIndex.popOver};
+  display: table;
```

Comment:
Why is a `table` display necessary OOC?

Reply:
Good question. I answered the same question from Dave. You can check here: https://code.amazon.com/reviews/CR-39859486

Basically speaking, `width: fit-content` does not work in Safari webView, which means it does not work in Mac Desktop Client. `display: table;` is an alternate which works exactly like `width: fit-content` and no hurt to web client.